source/install/cloud-changelog.md

```diff
@@ -4,12 +4,111 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
-- [Release 2022-10-27](#release-2022-10-27)
+
+## Release 2023-01-12
+
+### Highlights
+
+#### Message Priority and Acknowledgments
+ - Added [message priority labels](https://docs.mattermost.com/channels/message-priority.html) to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts (Professional).
+
+#### Global Drafts
+ - Added a centralized Drafts view for draft messages.
+ 
+#### Playbooks
+ - Added an option to run playbooks without creating a new channel every time in order to reduce the unnecessary overhead.
+ - In addition to the daily digest, users can now also view [a task inbox](https://docs.mattermost.com/playbooks/work-with-tasks.html#task-inbox) from the global header bar while in Playbooks.
+ 
+#### Calls
+ - [Audio calling and screen sharing](https://docs.mattermost.com/configure/calls-deployment.html) in channels is now generally available to all Mattermost customers.
+ - Updated [the keyboard shortcut](https://docs.mattermost.com/channels/keyboard-shortcuts-for-channels.html#calls-shortcuts) to start and join calls.
```

Comment:
Not sure if this is the right place but an important change was made to calls participants limits. The Cloud free plan will now support up to 8 participants per call in either group message, public or private channels (it was previously DMs only).

Reply:
Docs has captured this update via https://github.com/mattermost/docs/pull/6104 for the [Workspace Limits](https://docs.mattermost.com/onboard/mattermost-limits.html) product documentation page, and an extra mention in the changelog is a good idea for extra visibility.